assignment5/unknown_language.py

```diff
@@ -0,0 +1,55 @@
+'''
+Created on 07 July 2017
+
+@author: Eva
+'''
+
+def get_alphabet_adj(dictionary):
+    ''' Creates an alphabet given the dictionary of an alien language.
+        
+        Args:
+            dictionary: a list of words from the alien language given in 
+            lexicographic order
+        
+        Returns:
+            The alphabet of the dictionary in lexicographic order.
+    
+    '''
+    if not is_valid(dictionary):
+        raise ValueError("must be non-empty: dictionary")
```

Comment:
Just a thought - why not just return an empty alphabet?
I guess it's just a matter of personal style, but I feel like I would expect the function to return an empty list when the dictionary itself contains no words, and not raise an error :)